src/errors.js

```diff
@@ -13,4 +13,8 @@ https://www.algolia.com/users/sign_up/places.
 
 Or upgrade your 100,000 queries/month plan by contacting us at
 https://community.algolia.com/places/contact.html.`,
+  invalidCredentials: `The APP ID or API key provided is invalid.`,
+  invalidAppId: `Your APP ID is invalid. You must create a valid Places app first.
```

Comment:
What do you think of adding to the error message that Places APP IDs all start with `pl`?